subsys/bluetooth/controller/ll_sw/ull_adv_types.h

```diff
@@ -19,10 +19,15 @@ struct ll_adv_set {
 	uint8_t  rnd_addr[BDADDR_SIZE];
 	uint8_t  sid:4;
 	uint8_t  is_created:2;
+	uint16_t max_events;
+	uint16_t event_counter;
+	uint32_t remain_duration;
 #else /* !CONFIG_BT_CTLR_ADV_EXT */
 	uint16_t interval;
 #endif /* !CONFIG_BT_CTLR_ADV_EXT */
 
+	uint32_t next_random_delay;
```

Comment:
I do not see the need for this. Additionally you are adding overhead to legacy advertising feature and could complicate if conditional compiling extended advertising.

Reply:
I don't see another solution Asked Peter Hansen.
Keeping the code until we have another solution.